sync15-adapter/src/lib.rs

```diff
@@ -11,7 +11,6 @@ extern crate hyper;
 
 extern crate failure;
 
-#[macro_use]
 extern crate failure_derive;
```

Comment:
Do we still need the failure_derive crate explicitly? I think it's re-exported from failure now, we should be able to remove it here and from Cargo.tomls...